### PR TITLE
Implement additional parameters for internal link type

### PR DESCRIPTION
**Pull request by mkoitka**
This is extremely useful for tracking campaigns via Google Analytics or Piwik. Customers often wish for these tools to work together with their publicly announced short urls.

### CODE REVIEW

**Comment by mkoitka** [comment]
Any news/opinion on this?

File: Configuration/TCA/tx_urlforwarding_domain_model_redirect.php [hunk 1]
```diff
@@ -227,6 +229,13 @@
                 'default' => 302
             ]
         ],
+        'additional_parameters' => [
```
Comment (patrickbroens) [inline Configuration/TCA/tx_urlforwarding_domain_model_redirect.php]: This value should be added to ext_tables.sql as well

**Comment by mkoitka** [comment]
Oops, completely forgot about that. Thanks for reviewing!

**Comment by mkoitka** [comment]
As this was already fixed, is there anything else speaking against merging this?

**Pull request closed without merging**